models/classes/runner/time/TimerAdjustmentMapInterface.php

```diff
@@ -34,30 +34,32 @@ interface TimerAdjustmentMapInterface
     /**
      * Puts an increase to the map
      * @param string $sourceId
+     * @param string $type
      * @param int $seconds
      * @return TimerAdjustmentMapInterface
      */
-    public function increase(string $sourceId, int $seconds): TimerAdjustmentMapInterface;
+    public function increase(string $sourceId, string $type, int $seconds): TimerAdjustmentMapInterface;
 
     /**
      * Puts an decrease to the map
      * @param string $sourceId
+     * @param string $type
      * @param int $seconds
      * @return TimerAdjustmentMapInterface
      */
-    public function decrease(string $sourceId, int $seconds): TimerAdjustmentMapInterface;
+    public function decrease(string $sourceId, string $type, int $seconds): TimerAdjustmentMapInterface;
 
     /**
-     * Gets the calculated adjustment in seconds
+     * Gets the calculated total adjustments of all types stored for provided source ID in seconds.
      * @param string $sourceId
      * @return int
      */
     public function get(string $sourceId): int;
 
     /**
-     * Removes an entry specified by $sourceId
      * @param string $sourceId
```

Comment:
I would add a comment about what this function does, similar to `get(...)`

Reply:
Done.